restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -496,11 +497,11 @@
 	"webTriggerModalWebTriggerUrlHelp8": "módulo de enfoque",
 	"webTriggerModalWebTriggerUrlHelp9": "variables RVD cuando la aplicación se inicie",
 	"webTriggerModalAccessTokenLabel": "Token de acceso",
-	"webTriggerModalAccessTokenHelp1": "Proteja su URL de activación web con un token de acceso. Si esta Protect your Web Trigger URL with an access token. Si ya esta establecido, asegúrese de ingresar el mismo",
+	"webTriggerModalAccessTokenHelp1": "Proteja su URL de activación web con un token de acceso. Si ya esta establecido, asegúrese de ingresar el mismo",
 	"webTriggerModalAccessTokenHelp2": "token",
 	"webTriggerModalAccessTokenHelp3": "parámetros cuando se efectua la llamada HTTP. Dejarlo en blanco para permitir libre acceso.",
-	"webTriggerModalAccessTokenInput": "token de acceso",
-	"webTriggerMissingAccessTokenWarn": "el token de acceso esta vacío! Su activador solo podrá ser accesado utilizando una autenticación HTTP básica.",
+	"webTriggerModalAccessTokenInput": "Token de acceso",
+	"webTriggerMissingAccessTokenWarn": "El token de acceso esta vacío! Su activador solo podrá ser accesado utilizando una autenticación HTTP basic.",

```

Comment:
Grammar error found. Change for:
"webTriggerMissingAccessTokenWarn": "El token de acceso está vacío! Su activador sólo podrá ser accedido utilizando una autenticación HTTP básica.",